src/libs/ajax.js

```diff
@@ -356,6 +356,12 @@ export const DataSet = {
     const url = `${await getApiUrl()}/api/dataset/study/${studyId}`;
     return await axios.put(url, studyObject, Config.multiPartOpts());
   },
+
+  getDatasetByDatasetIdentifier: async datasetIdentifier => {
+    const url = `${await getApiUrl()}/api/tdr/${datasetIdentifier}`;
+    const res = await fetchOk(url, Config.authOpts());
```

Comment:
The fetchOK method will only return if the response is successful. In the case of a 404, for instance, it errors out and the page doesn't display anything. Can we add some logic in `DatasetStatistics` to capture that? You could call `fetchAny` instead and manually handle the non-success case in the UI. Separate from that, we've been trying to migrate to axios (one of our many transitions) so it might be cleaner using that approach.